src/test/wiki/FitNesseRoot/HsacPlugin/AllArgTableTemplateTest.wiki

```diff
@@ -59,4 +62,23 @@ All arg template also allows you to define more input parameters in input table
 
 |test |
 |P1|P2|
-|a |b |
\ No newline at end of file
+|a |b |
+
+All arg table also sets 2 automatic variables:
+ * '''!-@{CURRENT}-!''' (or '''!-@{SCENARIO_TESTS_CURRENT}-!''') which keeps row number of that is currently executed from fundamental decision table
+ * '''!-@{TOTAL}-!''' (or '''!-@{SCENARIO_TESTS_TOTAL}-!''') which keeps number of tests rows in fundamental decision table
```

Comment:
The word 'fundamental' does not seem quite appropriate here, in http://fitnesse.org/FitNesse.UserGuide.WritingAcceptanceTests.SliM.ScenarioTable 'invoking' is used. Would that be suitable here?

And instead of 'keeps' I recommend 'contains'.